app/views/admin/videos/new.html.haml

```diff
@@ -0,0 +1,22 @@
+%section.admin_add_video
+  .container
+    .row
+      .col-md-10.col-md-offset-1
+        =bootstrap_form_for [:admin, @video], layout: :horizontal, label_col: "col-sm-3", control_col: "col-sm-6" do |f|
+          %ul.nav.nav-tabs
+            %li
+              %a(href="") Recent Payments
+            %li.active
+              %a(href="") Add a New Video
+          %br
+          %fieldset
+            =f.text_field :title
+            =f.select :category_id, Category.all.collect {|c| [c.name, c.id]}
+            =f.text_area :description, rows: '8'
+            =f.file_field :large_cover, class: "form-control"
```

Comment:
looks like i lose the surrounding text field if i don't give it that class :confused:
![image](https://cloud.githubusercontent.com/assets/6085500/9426591/c0f90d32-498f-11e5-896e-23d636552c34.png)